x-team/xp-ui
# Copy email to clipboard on click

**Release Type:** *Non-Breaking Feature* 

Fixes https://x-team-internal.atlassian.net/browse/XP-2749

## Description



## Checklist

**Before submitting a pull request,** please make sure the following is done:



## Related PRs

XP-2749-copy-email-to-clipboard-on-click | [XP](https://github.com/x-team/xp/pull/1863)

## Steps to Test or Reproduce

Download Branch
Install new package and flow definitions by running `npm install`
run npm`start`
Under UI components GenericCopyToClipboard component should appear

## Impacted Areas in Application

Another new component

## Screenshots

![demo2](https://user-images.githubusercontent.com/25016886/55851455-44078200-5b1e-11e9-8593-17290eae6494.gif)
